ui/page/wallet_page.go

```diff
@@ -433,21 +427,27 @@ func (pg *WalletPage) walletSection(gtx layout.Context) layout.Dimensions {
 						})
 					}),
 					layout.Rigid(func(gtx C) D {
-						return layout.Inset{Top: values.MarginPadding10, Bottom: values.MarginPadding10}.Layout(gtx, func(gtx C) D {
-							return layout.Flex{Axis: layout.Horizontal, Alignment: layout.Middle}.Layout(gtx,
-								layout.Rigid(func(gtx C) D {
-									return layout.Inset{
-										Right: values.MarginPadding10,
-										Left:  values.MarginPadding38,
-									}.Layout(gtx, listItem.addAcctBtn.Layout)
-								}),
-								layout.Rigid(func(gtx C) D {
-									txt := pg.Theme.H6(values.String(values.StrAddNewAccount))
-									txt.Color = pg.Theme.Color.Gray
-									return txt.Layout(gtx)
-								}),
-							)
+						return pg.clickableRow(gtx, pg.listItems[i].addAcctClickable, func(gtx C) D {
```

Comment:
Remove redundant wrapper function
```suggestion
						return decredmaterial.Clickable(gtx, pg.listItems[i].addAcctClickable, func(gtx C) D {
```